src/test/java/fi/okm/jod/yksilo/service/ToimenkuvaServiceTest.java

```diff
@@ -28,7 +31,7 @@
 import org.springframework.test.context.jdbc.Sql;
 
 @Sql("/data/osaaminen.sql")
-@Import(ToimenkuvaService.class)
+@Import({ToimenkuvaService.class, YksilonOsaaminenService.class})
 class ToimenkuvaServiceTest extends AbstractServiceTest {
```

Comment:
It might be good to have a test where toimenkuva endDate is null. Those could be also validation tests for ValidInterval

Reply:
Good suggestion, will add a test for the validation logic.